tools/append_hashes.py

```diff
@@ -0,0 +1,102 @@
+'''
+Recurrent response file traverser.
+
+Extends each file name from a file list with it's full path hash and copies original file to hashed file.
+Response files (i.e. whose name start with @ and who contain a list of other files) are processed recursively.
+
+Modifies response files content with hashed file names.
```

Comment:
please explain why we use hashing in these docs

Reply:
Done with commit amendment.